pyproject.toml

```diff
@@ -35,3 +35,20 @@ doc = ["sphinx"]
 "Homepage" = "https://github.com/USEPA/harmonize-wq"
 "Documentation" = "https://usepa.github.io/harmonize-wq/"
 "Bug Tracker" = "https://github.com/USEPA/harmonize-wq/issues"
+
+
+[tool.ruff.lint]
+select = [
+    "E",
+    "F",
+    "W",
+    "I"
+]
```

Comment:
**suggestion**:  You might want to take a look at [the list of rules available](https://docs.astral.sh/ruff/rules/). This configuration is equivalent to [flake8](https://flake8.pycqa.org/en/latest/) + [isort](https://pycqa.github.io/isort/), but we could want a different set of rules for `harmonize_wq`

Rules can also be ignored:
- entirely
- at the file level
- for a specifc line using a comment `# noqa: E722` for instance